etcd/client.go

```diff
@@ -11,6 +11,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// This package provides a simple DAO to ease the interaction with ETCD. It also provides a lock system.
```

Comment:
In the go doc, you should start by `Package etcd ....`.

Reply:
And I think you can remove the extra line between the comment and the name of the package :)